organization/_posts/2018/2018-11-07-packaging.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "HSF Packaging Group Meeting #29, November 7, 2018"
+layout: meetings
+---
+
+# {{page.title}}
+
+Agenda
+=======
+[*https://indico.cern.ch/event/766022/*](https://indico.cern.ch/event/766022/)
+
+Participants: Graeme Stewart, Pere Mato, Patrick Gartung, Ben Morgan, James Amundson, Emil Obreshkov, Guilherme Amadio, Martin Ritter, Chris Green, Oana Boeriu
+
+Introduction
+============
+- Next HSF workshop is going to be 18-22 March 2019 at JLab
+  - Will be held jointly with WLCG and OSG
+  - More information to follow, but save the date
+
+
+SFT Spack Update
+=================
+- SFT group looking at other approaches to stack building, Spack continues as most promising tool/approach.
+- New features coming (upstream) that help
+  - Chains
+  - Fuzzy concretization
+- Synergy with HEP/Science communities
+- Goal of LCG/Spack
+  - Fairly complete current LCG release, built with Spack
+- Use this as “base” for
+  - Production runtime
+  - Developer buildenv
+- Initially targeting CentOS7 + GCC 8.2
+- Package sets:
+  - Gaudi+Externals (broad, but not huge, number)
+  -FCC Software (already using Spack, so this exercises Spack Chains, lower chain being Gaudi)
```

Comment:
I think you need a space here after the dash to make the bulleting work correctly